### PR TITLE
Fix a couple of typos in builtin and collect some forgotten strings in noise/vector settings dialog

### PR DESCRIPTION
**Pull request by Wuzzy2**
This PR fixes a couple of typos in builtin.
This PR also collects some forgotten strings in noise/vector settings dialog to make them available for translation.

### CODE REVIEW

**Comment by Wuzzy2** [comment]
Also, the description of `humid_rivers` flag was as follows:

> #    'humid_rivers': Increases humidity around rivers and where water pools.

This is bad grammar and confusing. Am I correct in assuming the sentence is correct when I remove the word “where”? @paramat

File: builtin/mainmenu/dlg_settings_advanced.lua [hunk 1]
```diff
@@ -649,11 +649,11 @@ local function create_change_setting_formspec(dialogdata)
 
 		height = height + 0.3
 		formspec = formspec
-				.. "field[0.3," .. height .. ";3.3,1;te_x;X;" -- X
+				.. "field[0.3," .. height .. ";3.3,1;te_x;"..fgettext("X")..";" -- X
```
[the remark this answers is not in the document]
Comment (nerzhul) [reply builtin/mainmenu/dlg_settings_advanced.lua]: not translatable

**Comment by nerzhul** [comment]
it will require a new translation merge with weblate when merged. Tell me

**Comment by Wuzzy2** [comment]
Yes, strings will be changed, so a Weblate update is mandatory.

@paramat: These are user-facing strings. Also note that some languages don't use ASCII script. Basically this is just to be “safe”. Most translatos can just keep it as-is, but this gives freedom for translators who need to change this for some reason.

**Comment by Wuzzy2** [comment]
I don't think I have seen standalone (!) translatable X/Y/Z anywhere else. So is the first time.

File: builtin/mainmenu/dlg_settings_advanced.lua [hunk 2]
```diff
@@ -632,11 +632,11 @@ local function create_change_setting_formspec(dialogdata)
 		end
 		-- Flags
 		formspec = table.concat(fields)
-				.. "checkbox[0.5," .. height - 0.6 .. ";cb_defaults;defaults;" -- defaults
+				.. "checkbox[0.5," .. height - 0.6 .. ";cb_defaults;"..fgettext("defaults")..";" -- defaults
```
Comment (paramat) [inline builtin/mainmenu/dlg_settings_advanced.lua]: Inconsistent use of spaces around '..'
Overly-long line (limit is 90 with tab size 4)

File: builtin/mainmenu/dlg_settings_advanced.lua [hunk 3]
```diff
@@ -632,11 +632,11 @@ local function create_change_setting_formspec(dialogdata)
...
 				.. tostring(flags["defaults"] == true) .. "]" -- to get false if nil
-				.. "checkbox[5," .. height - 0.6 .. ";cb_eased;eased;" -- eased
+				.. "checkbox[5," .. height - 0.6 .. ";cb_eased;"..fgettext("eased")..";" -- eased
```
Comment (paramat) [inline builtin/mainmenu/dlg_settings_advanced.lua]: Inconsistent use of spaces around '..'
Possible overly-long line (limit is 90 with tab size 4)

File: builtin/mainmenu/dlg_settings_advanced.lua [hunk 4]
```diff
@@ -632,11 +632,11 @@ local function create_change_setting_formspec(dialogdata)
...
 				.. tostring(flags["eased"] == true) .. "]"
-				.. "checkbox[5," .. height - 0.15 .. ";cb_absvalue;absvalue;" -- absvalue
+				.. "checkbox[5," .. height - 0.15 .. ";cb_absvalue;"..fgettext("absvalue")..";" -- absvalue
```
Comment (paramat) [inline builtin/mainmenu/dlg_settings_advanced.lua]: Inconsistent use of spaces around '..'
Overly-long line (limit is 90 with tab size 4)

File: builtin/mainmenu/dlg_settings_advanced.lua [hunk 5]
```diff
@@ -649,11 +649,11 @@ local function create_change_setting_formspec(dialogdata)
 
 		height = height + 0.3
 		formspec = formspec
-				.. "field[0.3," .. height .. ";3.3,1;te_x;X;" -- X
+				.. "field[0.3," .. height .. ";3.3,1;te_x;"..fgettext("X")..";" -- X
 				.. core.formspec_escape(v3f[1] or "") .. "]"
-				.. "field[3.6," .. height .. ";3.3,1;te_y;Y;" -- Y
+				.. "field[3.6," .. height .. ";3.3,1;te_y;"..fgettext("Y")..";" -- Y
```
Comment (paramat) [inline builtin/mainmenu/dlg_settings_advanced.lua]: Inconsistent use of spaces around '..'
Possible overly-long line (limit is 90 with tab size 4)

File: builtin/mainmenu/dlg_settings_advanced.lua [hunk 6]
```diff
@@ -649,11 +649,11 @@ local function create_change_setting_formspec(dialogdata)
...
 				.. core.formspec_escape(v3f[2] or "") .. "]"
-				.. "field[6.9," .. height .. ";3.3,1;te_z;Z;" -- Z
+				.. "field[6.9," .. height .. ";3.3,1;te_z;"..fgettext("Z")..";" -- Z
```
Comment (paramat) [inline builtin/mainmenu/dlg_settings_advanced.lua]: Inconsistent use of spaces around '..'
Possible overly-long line (limit is 90 with tab size 4)

**Comment by paramat** [comment]
I guess this is a blocker as those need to be translatable.

**Comment by paramat** [comment]
Thanks no objections.

**Pull request merged**